.circleci/config.yml

```diff
@@ -79,6 +235,19 @@ workflows:
           context:
             - circleci-user
             - tier-1-tap-user
+      - build_mongo_4_4:
+          context:
+            - circleci-user
+            - tier-1-tap-user
+          requires:
+            - build
+      - build_mongo_5_0:
+          context:
+            - circleci-user
+            - tier-1-tap-user
+          requires:
+            - build
```

Comment:
I think this can be deleted since it already depends on `build_mongo_4_4`

Reply:
Ok